### DISCUSSION TITLE
Incorrect or unused strings in C3 vanilla: fixable in Augustus?

### DISCUSSION THREAD

**Discussion by Ouaz** [Keriew/augustus#480] [General]
Hi!

There are some incorrect or unused strings in C3 vanilla.

Can it be fixed in Augustus, or does it require too much work ? (like creating again the strings in Augustus language file) 

---------------------------------------------------------------------------

**1) Incorrect strings are displayed for formations in Legion Panel:**

![formation_wrong_desc](https://user-images.githubusercontent.com/11946570/131404154-1a9044e0-ae1b-49b2-a269-30cad1810341.png)

Tortoise Formation has the description of the Column Formation, and Column Formation has the description of the Line Formation.
The Tortoise Formation strings exist in the language file:

_"Tortoise formation"_  `<group id="138"><string id="12">`
_"A highly defensive formation. Virtually impervious to missile fire."_ `<group id="138"><string id="18">`

---------------------------------------------------------------------------------------------

**2) Always the same string about the warehouseman, when right-cliking a warehouse**

![warehouseman_status](https://user-images.githubusercontent.com/11946570/131405156-4117f984-fa73-452d-95da-65c914f30b0f.png)

Either a warehouseman is currently delivering something or returning from a delivery, the same string is always displayed saying that _"Our cart is returning from a delivery."_
Once again, there's a string for a delivering warehouseman in the language file:

_"Our cart is taking goods elsewhere."_ `<group id="99"><string id="16">`

---------------------------------------------------------------

**3) Description strings not displayed for some buildings**

Farms and raw material buildings are the only ones to have both their current status and description displayed in their panels:

![building_desc](https://user-images.githubusercontent.com/11946570/131406243-2ac64fd1-446a-4988-b4e7-719a72cae4f4.png)

All other buildings having employees, and therefore an activity status (workshops, school, theater, market, etc.), have not their descriptions displayed:

![building_no_desc](https://user-images.githubusercontent.com/11946570/131406434-431e70a3-a095-4802-b2d5-6f7ab33bd785.png)

Although there are description strings in the language file for each building in the game.

For exemple, for Pottery Workshop:

_Here, potters mould clay into pots that citizens use for storing commodities. Trade pottery, or let your markets distribute it so that people can build better housing._  `<group id="126"><string id="1">`

----------------------

Point 1) and 2) seems to be a bug, but maybe point 3) was intended by the original devs (laziness? to save space?), who knows... I find these descriptions pretty good, and it adds consistency to the UI.

Thanks in advance for the answer!

Ouaz

**Comment by Keriew** [comment]
1. This seems to be correct, though it may look confusing (and doesn't match the help text). There's a separate formation in the code for the tortoise formation and it's unused, but all the meaningful interactions that one would expect from it (e. g. missile resistance) are tied to the column formation instead. I suspect it's either a dropped feature, or the formation overlapped too much and were merged at some point in the development.

2. We've actually fixed that already! You can check it in the unstable.

3. I think that'd make a good addition.

**Comment by Keriew** [comment]
3. Added in e6a2a58 and b085b10.

**Reply by Ouaz** [reply]
Nice work! Thanks!
